taqueria-protocol/taqueria-protocol-types.ts

```diff
@@ -82,7 +82,7 @@ const commandType: unique symbol = Symbol()
 export class Command extends StringLike {
     [commandType]: void
     static create(value: string) : Command | undefined {
-        if (value.match(/^([A-Za-z-_ ]+ ?)((\[.+\] ?)|(\<.+\>) ?)*$/)) {
+        if (value.match(/^([A-Za-z-_ ]+ ?)((\[[^\]\[\n]*\] ?)|(<[^<>\n]+>) ?)*$/)) {
```

Comment:
is this an unrelated change?

Reply:
Yes, this change was added to show what happens when an issue that CodeQL finds is fixed. Removing this change will cause the workflow to fail which I think is acceptable. There are also some other options to dismiss the CodeQL findings that are shown here:

![image](https://user-images.githubusercontent.com/29157965/160656526-a3ae7b1c-9ffe-4eef-aa8f-15a00bb3710c.png)

Reply:
I removed the change